ext/pdo/pdo_stmt.stub.php

```diff
@@ -69,4 +69,5 @@ public function getIterator(): Iterator {}
 
 final class PDORow
 {
+    public ?string $queryString;
```

Comment:
I think we should also declare the type for PDOStmt. And as far as I can tell, the one on PDOStatement can't be null.

Reply:
I'm not exactly sure how `PDORow::$queryString` gets its value. I found `get_lazy_object()` where the statement is preserved in the row (`row->stmt = stmt;`), and then `row_get_properties_for()` where property values are fetched from `row->stmt`. Then can we conclude that `PDORow::$queryString` has the same value as its `PDOStatement` counterpart? Or I misunderstood something (e.g. I don't know much about the purpose of the `get_properties_for` handler)?

Reply:
I'd also be ok go with a pragmatic approach: document the intended type (not considering the implicit null default value). I think this is similar what did with the parameters in a few cases  - so that at least we can have meaningful documentation.

If you also think it makes sense, I'll conduct an RFC where the internal property handling could be made more consistent and typed (by using types and by making write handler respect the strict_types directive). What do you think? Is it possible to do in a minor release?

Reply:
Yes, I think this is correct. So (per above comment) this one doesn't need to be nullable either. However, you might want to move the check inside row_prop_read a bit higher to make sure that even if there is a column that is called `queryString`, you will not get the column value (which might have different type).